boards/DCM/Src/App/App_Imu.c

```diff
@@ -0,0 +1,44 @@
+#include <assert.h>
+#include <stdlib.h>
+
+#include "App_Imu.h"
+
+struct Imu
+{
+    bool (*update_data)(void);
+    bool (*is_data_valid)(void);
+    struct ImuData *(*get_imu_data)(void);
```

Comment:
This should be changed after the Io code is fixed

Reply:
done